protostar/cairo_testing/cairo1_test_collector.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, cairo_path: list[str]):
     def collect_cairo1_tests_and_cache_outputs(
         self,
         file_path: Path,
-    ) -> List[str]:
+    ) -> List[Tuple[str, cairo1.AvailableGas]]:
```

Comment:
why don't we use `tuple`?

Reply:
I just got used to pre Python 3.9 syntax, it's used everywhere in this module as well

Reply:
From what I discussed with others quite a while ago, we should move away from using the types from `typing` and use built-ins wherever possible. 


> it's used everywhere in this module as well

Not true, it is not used in `cairo_test_execution_state.py`.

I think we should use the built-ins when changing the code / adding a new code.

Reply:
Done